gTea/app/src/main/java/com/nechaieva/gtea/utils/Normalizer.java

```diff
@@ -1,4 +1,20 @@
-package com.nechaieva.gtea;
+package com.nechaieva.gtea.utils;
 
+/**
+ * A class used to normalize strings for the order.
```

Comment:
What about being a bit more precise on the expected behavior, e.g., what it does?